Simplenote/SimplenoteAppDelegate.m

```diff
@@ -251,6 +258,34 @@ - (void)hookWindowNotifications
     [nc addObserver:self selector:@selector(handleWindowDidResizeNote:)             name:NSWindowDidResizeNotification              object:self.window];
 }
 
+- (void)handleGetURLEvent:(NSAppleEventDescriptor *)event withReplyEvent:(NSAppleEventDescriptor *)replyEvent
+{
+    NSString *urlString = [[event paramDescriptorForKeyword:keyDirectObject] stringValue];
+    NSURL *url = [NSURL URLWithString:urlString];
+    
+    if (![[url host] isEqualToString:@"auth"]) {
+        return;
+    }
+    
+    if ([WPAuthHandler isWPAuthenticationUrl: url]) {
+        if (self.simperium.user.authenticated) {
+            // We're already signed in
+            [[NSNotificationCenter defaultCenter] postNotificationName:SPSignInErrorNotificationName
+                                                                object:nil];
+            return;
+        }
+        
+        NSString *plistPath = [[NSBundle mainBundle] pathForResource:@"config" ofType:@"plist"];
+        NSDictionary *config = [[NSDictionary alloc] initWithContentsOfFile:plistPath];
```

Comment:
Since this snippet is (at least) in 3 spots (Login, AppDelegate 2x), how about if we do something like....

```
@objc class Secrets: NSObject {

     static let shared = Secrets()

     private let config: [String: Any]
     private init() {
          config = // .. load code!
     }
     var simperiumAppID: String {
         return config["..."
      }
}
```

OR even better, implement the secrets replacement tool we've chatted over Slack. (But if we do so, it probably deserves a different PR, since it's out of the scope of this one!)